lib/src/logging/logging.cc

```diff
@@ -24,13 +51,16 @@ namespace facter { namespace logging {
 
     void setup_logging(ostream& os)
     {
-        // Initialize boost filesystem's locale to a UTF-8 default.
-        // Logging gets setup the same way via the default 2nd argument.
-#if !defined(__sun) || !defined(__GNUC__)
-        // Locale support in GCC on Solaris is busted, so skip it.
-        boost::filesystem::path::imbue(leatherman::locale::get_locale());
-#endif
-        lm::setup_logging(os);
+        try {
+            setup_logging_internal(os);
+        } catch (...) {
```

Comment:
`...` catches SEH on Windows, potentially masking things like access violations.  This should probably be `exception const&` to catch any C++ exceptions (although it'd be nice if it was more specific).